win32/Lib/regcheck.py

```diff
@@ -18,13 +18,15 @@
 
```

Comment:
This module has been marked as deprecated for the past 18 years. A comment set it for removal around "2009 or so".
Would it be better to just remove it?

However it is used by `win32/scripts/regsetup.py` (checks the registry after Python registration). I'm not sure about a modern use case for the `regsetup.py` script either.